cmd/job_run_list.go

```diff
@@ -0,0 +1,156 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/odpf/salt/log"
+	cli "github.com/spf13/cobra"
+	"google.golang.org/grpc"
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	pb "github.com/odpf/optimus/api/proto/odpf/optimus/core/v1beta1"
+)
+
+const (
+	jobStatusTimeout = time.Second * 30
+)
+
+type jobRunCmdArg struct {
+	projectName string
+	jobName     string
+	startDate   string
+	endDate     string
+}
+
+type dateRange struct {
+	start time.Time
+	end   time.Time
+}
+
+type getRequestFn func(jobRunCmdArg) (*pb.JobRunRequest, error)
+
+func jobRunListCommand(l log.Logger, defaultProjectName, defaultHost string) *cli.Command {
+	cmd := &cli.Command{
+		Use:     "runs",
+		Short:   "Get current job runs",
```

Comment:
How about we have command  `optimus job list-runs` it is more clear I believe
Short: Get Job run details

Reply:
done as part of latest commit